test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Order.cshtml

```diff
@@ -52,6 +52,9 @@
                 <option value="Credit">Credit</option>
                 <option value="Check">Check</option>
             </select>
+            <select>
+                <option value="Check">Check</option>
```

Comment:
This doesn't prove the `<option>` tag helpers are working correctly. (If it must be tested in a functional test ...) Need to have the "Check" `<option>` be selected just above and not selected here.

Suggest you change the `.cshtml` file so this one has `selected="selected"`. That attribute is cleared in the above `<select>` and won't be here.